server/controllers/refer/peopleController.test.ts

```diff
@@ -0,0 +1,68 @@
+import type { DeepMocked } from '@golevelup/ts-jest'
+import { createMock } from '@golevelup/ts-jest'
+import type { NextFunction, Request, Response } from 'express'
+import createError from 'http-errors'
+
+import PeopleController from './peopleController'
+import type { Person } from '../../@types/models/Person'
+import type PersonService from '../../services/personService'
+import { personFactory } from '../../testutils/factories'
+import personUtils from '../../utils/personUtils'
+
+describe('PeopleController', () => {
+  const token = 'SOME_TOKEN'
+  const request: DeepMocked<Request> = createMock<Request>({ user: { token } })
+  const response: DeepMocked<Response> = createMock<Response>({})
+  const next: DeepMocked<NextFunction> = createMock<NextFunction>({})
+
+  const personService = createMock<PersonService>({})
+
+  let peopleController: PeopleController
+
+  beforeEach(() => {
+    peopleController = new PeopleController(personService)
+  })
+
+  describe('show', () => {
+    let person: Person
+
+    beforeEach(() => {
+      person = personFactory.build({
+        name: 'Del Hatton',
+      })
+      personService.getPerson.mockResolvedValue(person)
+    })
+
+    it('renders the confirmPerson template with the found Person', async () => {
+      personService.getPerson.mockResolvedValue(person)
+
+      const requestHandler = peopleController.show()
+
+      await requestHandler(request, response, next)
+
+      expect(response.render).toHaveBeenCalledWith('referrals/confirmPerson', {
+        pageHeading: "Confirm Del Hatton's details",
+        personSummaryListRows: personUtils.summaryListRows(person),
+      })
+    })
+
+    it('throws an error if the person is not found', async () => {
+      const notFoundRequest: DeepMocked<Request> = createMock<Request>({
+        user: { token },
+        params: {
+          prisonNumber: 'NOTFOUND',
+        },
+      })
+
+      personService.getPerson.mockResolvedValue(null)
+
+      const requestHandler = peopleController.show()
+
+      const expectedError = createError(404, {
+        userMessage: `Person with prison number NOTFOUND not found.`,
```

Comment:
Per the comment on the service, I found the object in the second argument here not to impact the test